src/main/java/ui/listpanel/ListPanelCard.java

```diff
@@ -118,14 +124,23 @@ private Node getEventDisplay(TurboIssue issue, final int withinHours) {
      * @param comments
      * @return
      */
-    private static Node layoutEvents(Model model, TurboIssue issue,
-                                     List<TurboIssueEvent> events, List<Comment> comments) {
+    public static Node layoutEvents(Model model, TurboIssue issue,
+                                    List<TurboIssueEvent> events, List<Comment> comments) {
         VBox result = new VBox();
         result.setSpacing(3);
         VBox.setMargin(result, new Insets(3, 0, 0, 0));
 
-        // Events
+        // Label update events
+        TurboIssueEvent.createLabelUpdateEventNodes(
+                model,
+                events.stream()
+                      .filter(e -> e.isLabelUpdateEvent())
+                      .collect(Collectors.toList()))
+        .forEach(node -> result.getChildren().add(node));;
```

Comment:
The `forEach` is operatoring on each node returned by `TurboIssueEvent.createLabelUpdateEventNodes` so I was thinking that it should be at the same level. I will put the `forEach` right after the previous statement then.

The double semi-colon is not intended :-P